examples/AN00156_i2c_master_example/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.21)
+include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
+project(AN00156_i2c_master_example)
+
+option(XCOREAI "Build for xcore.ai instead of xcore-200" ON)
+
+set(APP_HW_TARGET XCORE-AI-EXPLORER)
```

Comment:
It recently occurred to me that we should be using XK-EVK-XU316 rather than XCORE-AI-EXPLORER since marketing seems to have abandoned this name (both still available in XTC 15.3)